content/en/blog/_posts/2023-04-11-volume-group-snapshot-alpha.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: blog
+title: "Kubernetes 1.27: Volume Group Snapshot Moves to Alpha"
+date: 2023-04-11T10:00:00-08:00
+slug: kubernetes-1-27-volume-group-snapshot-alpha
+---
+
+**Author:** Xing Yang (VMware)
+
+Volume Group Snapshot is introduced as an alpha feature in Kubernetes v1.27. The Volume Group Snapshot feature introduces a Kubernetes API that allows users to take a crash consistent snapshot of multiple volumes together. It uses a label selector to group multiple persistent volume claims for snapshotting. This new feature is only supported for CSI Volume Drivers.
```

Comment:
Please **manually wrap** the long lines for ease of review feedbacks and for ease of change tracking, especially for downstream localization teams.

e.g.
```suggestion
Volume group snapshot is introduced as an Alpha feature in Kubernetes v1.27.
This feature introduces a Kubernetes API that allows users to take a crash consistent
snapshot for multiple volumes together. It uses a label selector to group multiple
PersistentVolumeClaims for snapshotting.
This new feature is only supported for CSI volume drivers.
```

Reply:
Please actually wrap the text.

Reply:
Done